src/components/compound/DlTable/DlTable.vue

```diff
@@ -516,7 +516,6 @@
                                 v-if="showRowActions"
                                 key="visibleColumnsSlot"
                                 no-tooltip
-                                style="width: 25px"
```

Comment:
This is the row-actions, do we not want it there?

Reply:
@advaa1234 this is actually the fix. It limited the actions column to be too small. Now it's dynamic.